plugin/src/main/kotlin/com/ritense/plugin/repository/PluginProcessLinkRepository.kt

```diff
@@ -40,23 +39,23 @@ class PluginProcessLinkRepository(
     fun findByProcessDefinitionIdAndActivityIdAndActivityType(
         processDefinitionId: String,
         activityId: String,
-        activityType: ActivityType
+        activityType: ActivityTypeWithEventName
     ) =
         pluginProcessLinkRepositoryImpl.findByProcessDefinitionIdAndActivityIdAndActivityType(
             processDefinitionId,
             activityId,
-            activityType.toActivityTypeWithEventName()
+            activityType
         )
 
     fun findByPluginConfigurationIdAndActivityIdAndActivityType(
         pluginConfigurationId: PluginConfigurationId,
         activityId: String,
-        activityType: ActivityType
+        activityType: com.ritense.processlink.domain.ActivityTypeWithEventName
```

Comment:
Instead of having the package here, you could make an alias for `com.ritense.processlink.domain.ActivityTypeWithEventName`, like `ActivityTypeWithEventNameEntity`.

Reply:
Changed